p4-16/spec/grammar.mdk

```diff
@@ -383,6 +385,13 @@ tupleKeysetExpression
     ;
 // END:tupleKeysetExpression
 
+// BEGIN:optTrailingComma
+optTrailingComma
+    : /* empty */
+    | ","
+    ;
+// END:optTrailingComma
+
```

Comment:
No change in the definition of optTrailingComma -- simply moving its definition earlier in grammar.mdk to match the order that p4c defines these non-terminals relative to each other, so `diff` gives fewer differences.